docs/import/create_score_asset.js

```diff
@@ -109,9 +115,9 @@ function combineWithSnap(feature, snapKey, totalKey) {
         snapFeature.get(censusGeoidKey),
         blockGroupTag,
         snapFeature.get(censusBlockGroupKey),
-        snapPopTag,
+        povertyHouseholdsTag,
```

Comment:
We're not actually displaying snap households in the table/popup right now but it makes sense to add it I guess.

Reply:
Easier than hiding, I agree.